desktop/libs/dashboard/src/dashboard/tests.py

```diff
@@ -32,6 +32,7 @@
 from dashboard.facet_builder import _round_number_range
 from dashboard.models import Collection2, augment_response
 from dashboard.controller import DashboardController
+from unittest.mock import Mock
```

Comment:
Will brake depending on the Python, to importer properly

Reply:
As I am not getting errors in py3 locally, so need to test on CI for every commit.
so I am doing testing now will send the final PR if all good.